crossdock/otel-agent-config.yml

```diff
@@ -4,15 +4,19 @@ receivers:
       host_endpoint: localhost:5778
       fetch_endpoint: jaeger-collector:14250
     protocols:
+      grpc:
+        disabled: true
       thrift_compact:
       thrift_binary:
 
 exporters:
+  jaeger_cassandra:
```

Comment:
Same as comment above.

Reply:
Collector config by default uses cassandra backed. In this case we want to disable C* storage because this deployment acts as agent.

Reply:
Ah ok - but when there is an opinionated default config for agent this won't be a problem, as you explained above.